app/models/event.rb

```diff
@@ -70,6 +70,14 @@ def attendee_count
     self.signups.count + self.external_signups.count
   end
 
+  def deadline_passed?
+    if deadline
+      deadline < Time.now
+    else
+      false
+    end
```

Comment:
```suggestion
   return unless deadline
   
   deadline < Time.now
```
Zou je ook nog false ipv nil kunnen returnen als je dat ergens nodig heb (maar nil is falsey dus zou goed moeten gaan)

Reply:
```suggestion
    deadline&.send("<", Time.now)
```
Ben ik zelf minder fan van omdat send best tricky kan zijn maar wel een coole one liner.